docs/RFCS/README.md

```diff
@@ -11,18 +11,23 @@ This is a lightweight design-document process that is inspired by the
 
 1. Copy `template.md` to a new file and fill in the details. Commit
    this version in your own fork of the repository or a branch.
-   
+
 2. Submit a pull request to add your new file to the main repository.
    Each RFC should get its own pull request; do not combine RFCs with
    other files.
-   
-3. Go through the PR review. If there is consensus to proceed with the
-   project, change the `Status` field of the document to
-   `in-progress`, update the `RFC PR` field, and merge the PR. If the
-   project is rejected, either abandon the PR or merge it with a
-   status of `rejected` (depending on whether the document and
+
+3. Go through the PR review. When the dust on the PR review has settled,
+   and if there is consensus to proceed with the project, announce that
+   the RFC is entering its Final Comment Period (FCP) using the method
+   de jour (https://forum.cockroachlabs.com/ at the time of this
+   writing). The FCP should last a minimum of two working days to allow
+   collaborators to voice any last-minute concerns. If there is still
+   consensus to proceed after the FCP, change the `Status` field of the
+   document to `in-progress`, update the `RFC PR` field, and merge the
+   PR. If the project is rejected, either abandon the PR or merge it
+   with a status of `rejected` (depending on whether the document and
```

Comment:
Might be worth mentioning that accepted RFCs can hit snags during implementation that could have been caught during the RFC review process. This is undesirable and something we should try to avoid, but hard to eliminate completely unless we make the RFC process overly onerous. For example, what happens if an expert on part of the system is on an extended vacation when an RFC is being reviewed?